pkg/apis/extensions/types.go

```diff
@@ -899,3 +899,45 @@ type NetworkPolicyList struct {
 
 	Items []NetworkPolicy `json:"items"`
 }
+
+// +genclient=true
+// +nonNamespaced=true
+
+// StorageClass describes the parameters for a class of storage for
+// which PersistentVolumes can be dynamically provisioned.
+//
+// StorageClasses are non-namespaced; the name of the storage class
```

Comment:
a) never say "etcd" in docs that will face a user.  It's a detail we don't want them thinking about.

b) this comment does not adequately describe storage classes.  Something a bit more robust perhaps?

```
StorageClass describes a named "class" of storage offered in a cluster.  Different classes might map to quality-of-service levels, or to backup policies, or to arbitrary policies determined by the cluster administrators.  Kubernetes itself is unopinionated about what classes represent.  This concept is sometimes called "profiles" in other storage systems.

The name of a StorageClass object is significant, and is how users can request a particular class.
```